src/main/java/converter/DataConverter.java

```diff
@@ -0,0 +1,8 @@
+package converter;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface DataConverter {
+    List<FruitTransaction> converterToTransaction(List<String> inputReport);
```

Comment:
Please follow the Java naming conventions for method names. They should be verbs and typically start with lower case. In this case, `convertToTransaction` would be a better name.